01week/javascripting/data-types.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+
+// Write a JavaScript program to display the current day and time.
+function CurrentDateTime() {
+  console.log(new Date());
```

Comment:
Logic is great on all of these, but you always want your function to return something. So this function should be 
`function CurrentDateTime() { 
    return new Date()
 }`
And then to call it and possibly display the date `console.log(CurrentDateTime())`. Apply to all functions below.